activestorage/app/models/active_storage/filename.rb

```diff
@@ -61,7 +59,7 @@ def sanitized
   end
 
   def parameters #:nodoc:
-    Parameters.new self
+    ActionController::DataStreaming::DispositionFilenameParameters.new(sanitized)
```

Comment:
Don't we need to require this file one? Also `action_controller` because it is not in Active Storage.

Reply:
I think we don't have to require because this file is under `app` and it's on Rails engine

Reply:
Can we avoid [internally] exposing this class entirely? AFAICS, this method's result is only used in: https://github.com/rails/rails/blob/b2eb1d1c55a59fee1e6c4cba7030d8ceb524267c/activestorage/lib/active_storage/service.rb#L125

(And that method seems to have more in common with the surrounding lines in `send_file_headers!`, too.)

Reply:
> Can we avoid [internally] exposing this class entirely?

Hmm, how can I avoid?

Reply:
I think Matthew’s suggesting removing the `parameters` method here and using the new class directly in  `ActiveStorage::Service#content_disposition_with`.

Reply:
gotcha

Reply:
I was thinking of something like `ActionController::ContentDisposition.format(:inline, filename: "hello.jpg") # => "inline; filename=[...]"`

So we still need to expose a constant (and following @rafaelfranca's suggestion to move it out of the DataStreaming namespace), but having it do the work internally, and just return a string. It just feels a bit odd to me that we're exposing the class _instance_, when all we want to do is call `to_s` on it.

Reply:
Cool, I'll do that